cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -0,0 +1,121 @@
+// CPU Utilization
+resource "aws_cloudwatch_metric_alarm" "cpu_utilization_too_high" {
+  count               = var.rds_create_high_cpu_alarm ? 1 : 0
+  alarm_name          = "rds-${data.aws_db_instance.civiform.id}-highCPUUtilization"
+  comparison_operator = "GreaterThanThreshold"
+  evaluation_periods  = var.rds_alarm_evaluation_period
+  metric_name         = "CPUUtilization"
+  namespace           = "AWS/RDS"
+  period              = var.rds_alarm_statistic_period
+  statistic           = "Average"
+  threshold           = var.rds_max_cpu_utilization_threshold
+  alarm_description   = "Average database CPU utilization is too high."
+
+  dimensions = {
+    DBInstanceIdentifier = data.aws_db_instance.civiform.id
+  }
+}
+
+resource "aws_cloudwatch_metric_alarm" "cpu_credit_balance_too_low" {
+  count               = var.rds_create_low_cpu_credit_alarm ? length(regexall("(t2|t3)", var.postgres_instance_class)) > 0 ? 1 : 0 : 0
```

Comment:
**question (blocking):** why are we only doing this when the instance type is t2 or t3?

Reply:
CPU credit balance metrics are only relevant for t type instances and some articles only mentioned t2 and t3, but it looks like t4 should be included too, so I added that and added this information into the alarm create boolean description. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/burstable-credits-baseline-concepts.html